modules/codegen/src/smithy4s/codegen/internals/SmithyToIR.scala

```diff
@@ -742,6 +738,19 @@ private[codegen] class SmithyToIR(model: Model, namespace: String) {
       Hint.PackedInputs
     case d: DeprecatedTrait =>
       Hint.Deprecated(d.getMessage.asScala, d.getSince.asScala)
+    case doc: DocumentationTrait =>
+      val memberDocs = shape
+        .members()
+        .asScala
+        .map(_.getTarget())
+        .map(shapeId => (shapeId.name, model.expectShape(shapeId)))
+        .map({ case (name, shape) =>
+          (name, shape.getTrait(classOf[DocumentationTrait]).asScala)
+        })
+        .collect({ case (name, Some(v)) => (name, v.getValue()) })
+        .toMap
```

Comment:
memberDocs seems to always come up as an empty `Map`, and I don't know why exactly :S

Reply:
I'll take a look

Reply:
yeah so I think it's just that we were looking at the target type's docs, see https://github.com/disneystreaming/smithy4s/pull/731/files#r1083577898

I pushed a commit to fix that, also managed to squeeze in a couple cleanups (e.g. got rid of the tailrec function in favor of `mkString`): d3234c7ba8827f2f89a4b421167680a67287e1c4

Reply:
realized the linesIterator can be empty, so another one... 0b3bf439ccd802d67151df1b768c1eebe12c3e23